### PR TITLE
Speedup GMLANUtils unit tests

### PR DESCRIPTION
**Pull request by polybassa**

### CODE REVIEW

**Review by KhazAkar — COMMENTED** [review]
Looks rather fine to me, but this unittest parameter is merely for speed-up unit testing?

**Comment by polydroi** [comment]
Yes, the parameter is for the unit test only

File: scapy/contrib/automotive/gm/gmlanutils.py [hunk 1]
```diff
@@ -178,7 +186,8 @@ def GMLAN_GetSecurityAccess(
             if resp is not None and resp.returnCode == 0x37 and retry:
                 if verbose:
                     print("RequiredTimeDelayNotExpired. Wait 10s.")
-                time.sleep(10)
+                if not unittest:
+                    time.sleep(10)
```
Comment (KhazAkar) [inline scapy/contrib/automotive/gm/gmlanutils.py:190]: Isn't there needed some tester present? I'm not GMLAN expert :)

**Comment by polydroi** [comment]
The handling of tester present is not part of this function. A user need to enters a diagnostic session aswell. Both need to be done with other utility functions. 

On February 21, 2022 8:48:34 PM GMT+01:00, "Damian Zaręba" ***@***.***> wrote:
***@***.*** commented on this pull request.
>
>
>
>> @@ -178,7 +186,8 @@ def GMLAN_GetSecurityAccess(
>             if resp is not None and resp.returnCode == 0x37 and retry:
>                 if verbose:
>                     print("RequiredTimeDelayNotExpired. Wait 10s.")
>-                time.sleep(10)
>+                if not unittest:
>+                    time.sleep(10)
>
>Isn't there needed some tester present? I'm not GMLAN expert :)
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/secdev/scapy/pull/3528#pullrequestreview-889055275
>You are receiving this because you commented.
>
>Message ID: ***@***.***>

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Review by KhazAkar — APPROVED** [review]
Taking response into account, Looks Good To Me :)

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**